docs/INSTALL.rst

```diff
@@ -0,0 +1,85 @@
+Installation
+============
+
+Conferatur requires Python_ version 3.5 or above.
+
+
+From PyPI (preferred)
+---------------------
+
+This is the easiest and preferred way of installing conferatur.
+
+TODO
+
+
+From source
+-----------
+
+1. Install Python_ 3.5 or above (latest stable version for your OS is preferred):
+
+   Use the guides available at `The Hitchhiker’s Guide to Python <https://docs.python-guide.org>`_
+
+    - `Installing Python 3 on Mac OS X <https://docs.python-guide.org/starting/install3/osx/>`_
+    - `Installing Python 3 on Windows <https://docs.python-guide.org/starting/install3/win/>`_
+    - `Installing Python 3 on Linux <https://docs.python-guide.org/starting/install3/linux/>`_
+
+2. Get the conferatur source code from github (assumes :code:`git` is installed on the system)
+
+   .. git clone https://github.com/ebu/ai-benchmarking-stt.git
+
+   .. code-block:: bash
+
+      git clone https://github.com/MikeSmithEU/ai-benchmarking-stt.git
+
+3. Install the package using :code:`pip`, this will also install all requirements
+
+   .. code-block:: bash
+
+      pip install 'ai-benchmarking-stt[api]'
```

Comment:
Actually it does (at least for me), it copies the source to the site-packages, it is however not a "developer" version linked to that code directory. I've updated the docs to first do `cd ai-benchmarking-stt` and then `pip install '.[api]'` to avoid confusion.